### DISCUSSION TITLE
FLOSS research thread

### DISCUSSION THREAD

**Discussion by igordsm** [Insper/open-dev#814] [General]
Choose theme andtime slot for Lesson 11

**Comment by cemmanuelsr** [comment]
Linus Torvalds and Linux - 27/Novembro

**Reply by igordsm** [reply]
Quem será sua dupla?

**Reply by cemmanuelsr** [reply]
@cicerotcv

**Comment by nataliaqmc** [comment]
RedHat - 22 de Novembro @hsalberti

**Comment by JorasOliveira** [comment]
Canonical and Ubuntu - 29/11 @Antonioarf

**Comment by jpgianfaldoni** [comment]
Larry Wall and Perl 13/11
